src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Missing header comment. All non trivial classes/methods should have javadoc format header comments.